applications/crossbar/src/modules/cb_local_provisioner_templates.erl

```diff
@@ -16,6 +16,7 @@
 %%% @contributors
 %%%   Jon Blanton
 %%%   Karl Anderson
+%%%   Pierre Fenoll
```

Comment:
Hahahaha! Yes, I put it there for you to notice and you did haha. (right *above* your name)